KalturaWowzaServer/src/com/kaltura/media/server/wowza/RecordingManager.java

```diff
@@ -49,6 +51,24 @@
 		private KalturaMediaServerIndex index;
 		private double appendedDuration = 0;
 		private boolean isLastChunk = false;
+
+		class AppendRecordingTimerTask extends TimerTask {
+			
+			protected String filePath;
+			protected boolean lastChunkFlag;
+			
+			public AppendRecordingTimerTask (String path, boolean lastChunk) {
+				filePath = path;
+				lastChunkFlag = lastChunk;
+			}
+
+			@Override
```

Comment:
@hilak, instead of implementing with TODO, remove the implementation and make the class abstract.